src/services/test_service.js

```diff
@@ -1,4 +1,6 @@
+const error = require('../storage/logs').error('medmorph-backend:test');
 module.exports = (req, res, next) => {
+  error('there was an error');
```

Comment:
Is the plan to delete this? I think its fine to keep it as a test but the text should reflect that 
```suggestion
  error('test error', 'Test Service: Test Notification');
```